package.json

```diff
@@ -4,20 +4,38 @@
   "private": true,
   "dependencies": {
     "electron": "^1.7.9",
+    "enzyme": "^3.1.0",
```

Comment:
All packages related to tests should be in dev part.

Reply:
Will be fixed in the next commit.